### PR TITLE
Added 2 methods to the Joint class:

### PR DESCRIPTION
**Pull request by Nagillimi**
**`.show()`** method sets the Joint class `image.visible` property true

**`.detach()`** method polls the children of Joint's `imageWrapper` THREE.Group property for a matching `.id` parameter from the passed `Object3D` type. If true, the method removes the previously attached child from the `imageWrapper` group.

Would you like me to:
- update the README to include these methods?
- create live examples for show/hide & attach/detach?

### CODE REVIEW

**Comment by boytchev** [comment]
Thanks --

Yes, changes in README and live examples would be welcome.

Meanwhile, I have two questions:

1. What are the purposes of  `.show()` and `.detach()`? Do you need them for some specific example, or you just want to have a complete API?

2. Have you checked whether `.detach()` could be written as:
```javascript
detach(image)
{
  this.imageWrapper.remove(image);
} // Joint.detach
```
My expectation is that if `image` is a child, it will be removed, and if it is not a child, nothing will be done.

-- Pavel

**Comment by Nagillimi** [comment]
Sorry for the delay!

Sounds good, I'll submit another PR in the coming weeks with some live demos showcasing show/hide and attach/detach, following your examples. 

As for your questions:

1. Mainly to have a full API yep. While working with this repo a lot, these were the two things I thought would benefit every user (as well as myself and my team).

2. I checked that originally yes. Though, I was working with buggy source code, so let me check again and I'll patch if necessary!

Ben
